lib/+ne7/+scanimage/Reader4.m

```diff
@@ -58,12 +60,21 @@
             end
         end
         
+        function f = get.is_functional(self)
+            if self.scanimage_version == 4
```

Comment:
That was supposed to be `1` always for scanimage 4. Updated.

Basically I couldn't easily find a way to tell structural vs functional apart for scan image 4 so disabling this feature.